searxng_extra/update/update_languages.py

```diff
@@ -113,13 +114,88 @@ def join_language_lists(engines_languages):
                         print("ERROR: %s --> %s" % (locale, exc))
                         locale = None
 
-                language_list[short_code]['countries'][lang_code] = {'country_name': country_name, 'counter': set()}
+                language_list[short_code]['countries'][lang_code] = {
+                    'country_name': country_name,
+                    'flag': '',
+                    'counter': set(),
+                }
 
             # count engine for both language_country combination and language alone
             language_list[short_code]['counter'].add(engine_name)
             if lang_code != short_code:
                 language_list[short_code]['countries'][lang_code]['counter'].add(engine_name)
-
+    flags = {
+        'ar': '🇸🇦',
+        'be': '🇧🇾',
+        'bg-BG': '🇧🇬',
+        'cs-CZ': '🇨🇿',
+        'da-DK': '🇩🇰',
+        'de': '🇩🇪',
+        'de-AT': '🇦🇹',
+        'de-CH': '🇨🇭',
+        'de-DE': '🇩🇪',
+        'el-GR': '🇬🇷',
+        'en': '🇬🇧',
+        'en-AU': '🇦🇺',
+        'en-CA': '🇨🇦',
+        'en-GB': '🇬🇧',
+        'en-IE': '🇮🇪',
+        'en-MY': '🇲🇾',
+        'en-NZ': '🇳🇿',
+        'en-US': '🇺🇸',
+        'es': '🇪🇸',
```

Comment:
I guess the reasoning can be applied to `es` : it represents the Spanish language whatever the region is.